sig-network/charter.md

```diff
@@ -0,0 +1,42 @@
+# SIG Network
+
+## Scope
+
+SIG Network is owning everything related to KubevVirt networking.
+The primary objective of the sig is to connect the Virtual Machine to the pod network/s,
+allowing migration and other advantages of virtualization in addition to maintaining services,
+port-forwarding, service meshes, and other Kubernetes capabilities.
+
+### In scope
+Some notable examples of features owned by SIG Network:
+- Masquerade, Bridge and Passt binding plugins for the primary network.
+- Bridge, SRIOV and Macvtap bindings for secondary networks.
+- VM networking lifecycle on creation, migration, hotplug/unplug, restart, stop etc.
+
+#### Code, Binaries and Services
+- Networking related API on VM, VMI, Instance types, Kubevirt CR, etc.
+- Core network binding infrastructure.
+- Network binding plugin infrastructure.
+- Examples of binding plugin CNIs and sidecars.
+- Virtual Machine/Virtual Machine Instance networking status management.
+- Domain networking configuration.
+- Unit and e2e tests
+
+### Out of scope
+- Node networking, it is managed by [kubernetes-nmstate](https://github.com/nmstate/kubernetes-nmstate).
+- Host to Pod connectivity, it is managed by external CNIs.
+
+## Roles and Organization Management
+
+This sig follows adheres to the Roles and Organization Management outlined in [OARP]
+
+### Additional responsibilities of Chairs
+
+There are some other projects managed by the SIG:
+- [Cluster Network Addon](https://github.com/kubevirt/cluster-network-addons-operator).
+- [kubernetes-nmstate](https://github.com/nmstate/kubernetes-nmstate).
+- [kubernetes-nmstate-policy](https://github.com/nmstate/nmpolicy).
+- [KubeMacPool](https://github.com/k8snetworkplumbingwg/kubemacpool).
+- [KubeSecondaryDNS](https://github.com/kubevirt/kubesecondarydns)
```

Comment:
maybe need to add please macvtap and dynamic-networks ?
what about OVS-CNI, bridge marker? (any other thing CNAO deploys and we forgot)

Reply:
added